Gemfile

```diff
@@ -3,7 +3,7 @@ source "https://rubygems.org"
 # You may use http://rbenv.org/ or https://rvm.io/ to install and use this version
 ruby ">= 2.6.10"
 
-gem "cocoapods", "~> 1.13"
+gem "cocoapods", "~> 1.14"
```

Comment:
Question: Should we commit the changes to the Podfile.lock file or not?
<img width="1025" alt="Screenshot 2024-06-19 at 10 00 35 PM" src="https://github.com/Expensify/App/assets/35371050/63fd865b-3caf-4b62-9caf-dc37a6ce7617">

Reply:
Generally speaking, I think it's fine not to commit this change.